.github/workflows/release.yml

```diff
@@ -61,12 +61,17 @@ jobs:
           - os-name: linux
             os: ubuntu-20.04
             bazel-config: release_linux
+            target-arch: X64
```

Comment:
This is redundant at present (all releases built on GitHub Actions are for x86), but I plan to build the Apple Silicon release here too some time after #3079 has landed. Open to alternative suggestions.

Reply:
It's pretty good as-is imo

Reply:
👍
Merged and seems to work correctly now, thanks for approving!